src/build123d/__init__.py

```diff
@@ -16,6 +17,7 @@
     "Align",
     "AngularDirection",
     "CenterOf",
+    "Direction",
```

Comment:
The enum `Direction` has been eliminated, why is it added back here? It causes the tests to fail.

Reply:
Apologies, this must have been carried over from an older version. I have removed it again and it looks like the tests are doing better again.